packages/footer/tests/site/index.html

```diff
@@ -0,0 +1,101 @@
+		</script>
+	<![endif]-->
+</head>
+<body class="uikit-body">
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
+	</noscript>
+
+	<h1>Test: footer</h1>
+
+	<footer class="uikit-footer" role="contentinfo">
+		<nav class="uikit-footer__navigation">
+			<ul class="uikit-footer__navigation__list">
+				<li class="uikit-footer__navigation__list-group">
+					<h2>Menu title</h2>
+
+					<ul>
+						<li><a href="#">Related link</a></li>
```

Comment:
I added the `link-lists` module to the import items in the `test.scss` and simplified the footer module markup.

There’s now a new class that needs to be added to `link-lists`: `--block` (and [the existing inline-links class needs to be edited](https://github.com/govau/uikit/issues/72)).